### DISCUSSION TITLE
FluxNews photo downloading and photo gallery support. Novo Theme

### DISCUSSION THREAD

**Discussion by MahatmaNil** [inspireui/support#5784] [New Feature/Ideas Requests]
Hello. I'm a photographer. Now I have connected my website to the FluxNews application. We need to add the ability for users to download images from the blog. You can also open images in photo gallery mode in vertical and horizontal formats when you rotate your mobile device.
My waterpress theme is "NovoTheme".
https://themeforest.net/item/novo-photography-wordpress-theme/20701463

And please, as many integrations as possible for this topic in FluxNews. Hot among wedding photographers.
![Slide 1](https://github.com/inspireui/support/assets/160998297/a42c18fb-1f7f-438d-a212-a070d21162e8)
![Slide 2](https://github.com/inspireui/support/assets/160998297/1d696880-04aa-446f-89b1-b13839a44173)

**Comment by JesseJohnsonWeeding** [comment]
The WordPress theme “Novo” is really popular among photographers. You need to add support for galleries and portfolios from this theme, and simply display these galleries as regular blog posts in the FluxNews application, and show photos in “grid” mode with the ability to enlarge and download. This new introduction will attract new users and of course it is necessary for the business of photographers. We are waiting for this feature.

**Comment by Arjunthakura** [comment]
We need the ability to display images in full screen. Click on it and scroll through the next images in full screen mode.

**Comment by Yan-Aleman** [comment]
Hello. Plz support the task #5800
